### PR TITLE
Absolute Paths

### PR DESCRIPTION
**Pull request by lgmugnier**
## Description of the change

Turns all of our relative paths into absolute paths.

## Testing

run 
```
npm install
npm run release
```

**backend branch:** 
**bzero branch:** 
**charts branch:** 

#### Ready to run system tests?

- [x] Yes

## Relevant release note information

Release Notes: No functional changes

## Related JIRA tickets

Relates to JIRA: CWC-2599

## Relevant HotFix if applicable

Relevant HotFix PR Number:

## Have you considered the security impacts?

Does this PR have any security impact?

- [ ] Yes
- [x] No

If yes, please explain:

Does this PR add a new/modifies the name/arguments of a `zli` command?

- [ ] Yes
- [x] No

If yes, please provide the docs/zli-reference-manual branch reflecting the changes:

### CODE REVIEW

**Review by lipmas — APPROVED** [review]
Working for me on debian linux was able to compile and run zli without issues. I tested running `npm run release` and even `npm run release-all` to build executables across all platforms. Auto import is also working for me in vscode

**Review by chad-cwc — APPROVED** [review]
I tested `npm run release-all` and auto import in VSCode on Fedora 26. Both work as expected. This is awesome. Thanks, Lucie.

File: src/handlers/service-account/create-service-account.handler.ts [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+import { Logger } from 'services/logger/logger.service';
+import { ConfigService } from 'services/config/config.service';
+import { cleanExit } from 'handlers/clean-exit.handler';
 import yargs from 'yargs';
-import { ServiceAccountHttpService } from '../../http-services/service-account/service-account.http-services';
-import { createServiceAccountArgs } from '../../handlers/service-account/create-service-account.command-builder';
+import { ServiceAccountHttpService } from 'http-services/service-account/service-account.http-services';
+import { createServiceAccountArgs } from 'handlers/service-account/create-service-account.command-builder';
 import fs from 'fs';
 import { ServiceAccountProviderCredentials } from '../login/types/service-account-provider-credentials.types';
```
Comment (johncmerfeld) [inline src/handlers/service-account/create-service-account.handler.ts:8]: Busted

Comment (lgmugnier) [reply src/handlers/service-account/create-service-account.handler.ts]: damn!!!!! ab13767

**Review by johncmerfeld — APPROVED** [review]
Looked over the files, and ran the below two commands:
```
fgrep -r "from '../" --exclude-dir bzero --exclude-dir webshell-common-ts --exclude-dir bin --exclude-dir dist --exclude-dir node_modules --include '*.ts'
fgrep -r "from './" --exclude-dir bzero --exclude-dir webshell-common-ts --exclude-dir bin --exclude-dir dist --exclude-dir node_modules --include '*.ts' 
```

Both came up empty! And auto-imports work to boot!!

**Review by tflpd — APPROVED** [review]
Tested auto-import from the main module and submodules, compiling and using the zli, everything works like a charm.
Also tested applying the config changes of #move-to-es6 since we will be eventually be moving to that module compiler, still worked great. Awesome work!

**Pull request merged**